src/components/Voting.jsx

```diff
@@ -0,0 +1,37 @@
+import { updateVotes } from "../utils/api"
+import { useState } from "react"
+
+const Voting = ({article_id, articleVotes}) => {
+    
+    const [voteCount, setVoteCount ] = useState(0)
+    console.log(articleVotes)
+
+    const upVote = (Id) => {
+        if(voteCount === 0) {
+            setVoteCount(1)
+            updateVotes(Id, 1)
+        } else if (voteCount === -1){
+            setVoteCount(0)
+            updateVotes(Id, 1)
+        }
+    }
+
+    const downVote = (Id) => {
+        if(voteCount === 0) {
+            setVoteCount(-1)
+            updateVotes(Id, -1)
```

Comment:
Just need error handling - at the moment if im offline and i click the button, it looks like its worked